packages/handler-graphql/src/responses.ts

```diff
@@ -11,18 +12,24 @@ const defaultParams: ErrorResponseParams = {
 };
 
 export class ErrorResponse {
-    data: any;
-    error: {
+    public readonly data: null;
+    public readonly error: {
         code: string;
         message: string;
-        data?: any;
+        data: any;
+        stack: string;
     };
+
     constructor(params: ErrorResponseParams) {
         this.data = null;
+
+        const debug = process.env.DEBUG === "true";
+
         this.error = {
             code: params.code || defaultParams.code,
             message: params.message || defaultParams.message,
-            data: params.data || defaultParams.data
+            data: params.data || defaultParams.data,
+            stack: debug ? params.stack : defaultParams.stack
```

Comment:
Not super problematic/important, but still, maybe we can be explicit here and assign `null` when `stack` property doesn't exist.

This is the reason I went with a separate `if` statement here in my previous comment:

```ts
// This part here ensures `stack` is either `string` or `null`.
let stack = defaultParams.stack;
if (debug && params.stack) {
  stack = params.stack;
}
```

In your case, we'd have `undefined` in the `this.error.stack` property if the `debug` flag is true and `params.stack` property doesn't exist.

Initially I went also with ternary operator ...

```ts
stack: debug ? params.stack || defaultParams.stack : defaultParams.stack
```

... but since it was a bit hard to read, I decided to put it into a separate `if` above the `this.error` assignment.

Does this make sense?